src/pytorch_lightning/loggers/wandb.py

```diff
@@ -303,6 +303,8 @@ def __init__(
         log_model: Union[str, bool] = False,
         experiment: Union[Run, RunDisabled, None] = None,
         prefix: str = "",
+        checkpoint_name: Optional[str] = None,
+        use_reference: bool = False,
```

Comment:
How is this used?

Reply:
I've removed the `use_reference` argument